GoInfoGame/GoInfoGameTests/quests/TactilePavingStepsTests.swift

```diff
@@ -44,13 +44,21 @@ final class TactilePavingStepsTests: XCTestCase {
     
 
     
-    func testStepsQuery() throws {
+    func testtactilePavingStepsQuery() throws {
+        
+        let fourYearsAgo = TestQuestUtils.olderThan(years: -4)
+        let eightYearsAgo = TestQuestUtils.olderThan(years: -8)
+        
+
         assertIsNotApplicable(element: TestQuestUtils.node(tags: ["" : ""]))
         assertIsNotApplicable(element: TestQuestUtils.way(tags: ["highway" : "steps", "access": "no"]))
         assertIsNotApplicable(element: TestQuestUtils.node(tags: ["highway": "residential"]))
-        assertIsApplicable(element: TestQuestUtils.way(tags: ["highway": "steps", "surface": "paved"]))
-        assertIsNotApplicable(element: TestQuestUtils.way(tags: ["highway": "steps", "tactile_paving": "yes", "tactile_paving:date": "2023-01-31"])) // tactile_paving older than 8 years
-        assertIsNotApplicable(element: TestQuestUtils.way(tags: ["highway": "steps", "tactile_paving": "partial", "tactile_paving:date": "2023-01-31"])) // tactile_paving older than 4 years
+        assertIsApplicable(element: TestQuestUtils.way(tags: ["highway" : "steps", "surface": "paved"]))
+        assertIsApplicable(element: TestQuestUtils.way(tags: ["highway" : "steps", "surface": "paved", "tactile_paving": "unknown"]))
+        assertIsApplicable(element: TestQuestUtils.way(tags: ["highway" : "steps", "surface": "paved", "tactile_paving": "incorrect"], timestamp: fourYearsAgo))
+        assertIsApplicable(element: TestQuestUtils.way(tags: ["highway" : "steps", "surface": "paved", "tactile_paving": "yes"], timestamp: eightYearsAgo))
```

Comment:
where are the negative cases for timestamps??

Reply:
added negative cases for timestamps